python/singa/image_tool.py

```diff
@@ -426,7 +426,33 @@ def random_crop(self, patch, inplace=True):
             return self
         else:
             return new_imgs
-
+        
```

Comment:
the random_size_crop should be similar to https://github.com/pytorch/vision/blob/master/torchvision/transforms.py#L241